partiql-plan/src/main/java/org/partiql/plan/OperatorRewriter.java

```diff
@@ -11,26 +11,26 @@
 import java.util.List;
 
 /**
- * Operator transform is an abstract base visitor which recursively rewrites an operator tree.
```

Comment:
(self-review) rename to `OperatorRewriter` to be consistent with our other tree-walking APIs which use `Rewriter`